content/spin/v2/rdbms-storage.md

```diff
@@ -90,22 +88,13 @@ const DB_URL = "mysql://root:@127.0.0.1/spin_dev"
  insert into test values (4,4);
 */
 
-export const handleRequest: HandleRequest = async function (request: HttpRequest): Promise<HttpResponse> {
-
-    Mysql.execute(DB_URL, "delete from test where id=?", [4])
-    Mysql.execute(DB_URL, "insert into test values (4,5)", [])
-    let test = Mysql.query(DB_URL, "select * from test", [])
+export async function handler(_req: Request, res: ResponseBuilder) {
+  let conn = Mysql.open(DB_URL);
+  conn.execute('delete from test where id=?', [4]);
+  conn.execute('insert into test values (4,5)', []);
+  let ret = conn.query('select * from test', []);
```

Comment:
A comment showing what this looks like (or code showing how to use it) would be very useful here.  Looking at the help I _think_ there's an array of rows, each of which is an object keyed by column name (a la JSON `{ "columns": ???, "rows": [{ "id": 4, "val": 5 }] }` - is that right?

(I am not sure about the columns.  The reference suggests each column has a name and an _array_ of data types.  But I think I must be reading that wrong.)

Reply:
That was a typo. I will fix it in the docs. It should just be the name and the type of the column.

Reply:
Fixed the ref docs.